include/inexor/vulkan-renderer/wrapper/descriptor_builder.hpp

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include <vulkan/vulkan_core.h>
+
+#include <cassert>
+#include <string>
+#include <vector>
+
+namespace inexor::vulkan_renderer::wrapper {
+
+class Device;
+class ResourceDescriptor;
+
+class DescriptorBuilder {
+    const Device &m_device;
+    const std::uint32_t m_swapchain_image_count{};
+
+    std::vector<VkDescriptorSetLayoutBinding> m_layout_bindings{};
+    std::vector<VkWriteDescriptorSet> m_write_sets{};
```

Comment:
```suggestion
    std::vector<VkDescriptorSetLayoutBinding> m_layout_bindings;
    std::vector<VkWriteDescriptorSet> m_write_sets;
```

Reply:
I did this for all members which are std::vectors.